.eslintignore

```diff
@@ -2,4 +2,434 @@ build/
 dist/
 node_modules/
 .snapshots/
-*.min.js
\ No newline at end of file
+*.min.js
+
+# js build files where a ts source file exists
```

Comment:
Ensure we ignore all the temp build files from eslint as they don't conform.

Reply:
Is this because we are compiling ts to js in place? Presumably once everything is in TS we can switch to using a glob for this?